tests/test_obo_reading.py

```diff
@@ -118,3 +119,15 @@ def test_parse_tag_line_backslashed_exclamation():
     tag, value, trailing_modifier, comment = parse_tag_line(line)
     assert tag == 'synonym'
     assert value == r'not a real example \!'
+
+def test_ignore_obsolete_nodes():
+    hpo = obonet.read_obo("http://purl.obolibrary.org/obo/hp.obo")
+    nodes = hpo.nodes(data=True)
+    assert "HP:0005549" not in nodes
+
+def test_presence_of_obsolete_nodes():
+    hpo = obonet.read_obo("http://purl.obolibrary.org/obo/hp.obo", ignore_obsolete=False)
+    nodes = hpo.nodes(data=True)
+    assert "HP:0005549" in nodes
+    node = nodes['HP:0005549']
+    assert node['is_obsolete'] == 'true'
```

Comment:
Nice tests. 

`hp.obo` looks to be 6.7M, so for the purpose of keeping these tests speedy, we might want to test on a simple ontology that we store locally. See examples in [`tests/data`](https://github.com/dhimmel/obonet/tree/master/tests/data). This also protects us from future test failure if something changes at that URL.

One easy way might be to add the following to `tests/data/brenda-subset.obo` ([source](https://github.com/BRENDA-Enzymes/BTO/blob/6e1699e28c254389a149ff5d3026c3e204f02a22/bto.obo)):

```
[Term]
id: BTO:0000311
name: culture filtrate
comment: Added as synonym to culture fluid.
is_obsolete: true
```